librz/core/project.c

```diff
@@ -38,22 +42,40 @@ RZ_API RzProjectErr rz_project_save(RzCore *core, RzProject *prj, const char *fi
 }
 
 RZ_API RzProjectErr rz_project_save_file(RzCore *core, const char *file) {
+	char *tmp_file;
+	int err_mkstemp = rz_file_mkstemp("svprj", &tmp_file);
```

Comment:
please note that in theory the security guarantees of mkstemp come from the fact that you use the file descriptor returned by it, that is `err_mkstemp` in this case. However, we would need to change few APIs to actually use directly the descriptor instead of taking a file, so... Please close at least the file descriptor returned by this function.